### PR TITLE
Catch up with Makepad's theming changes

### PR DESCRIPTION
**Pull request by joulei**

### CODE REVIEW

**Comment by joulei** [comment]
Oops, my local `dev` was slightly behind upstream. 
I'm in the middle of catching up the changes with the latest `dev`.

File: src/chat/chat_panel.rs [hunk 1]
```diff
@@ -19,20 +19,21 @@ use super::{
...
+    use crate::shared::widgets::*;
+    use crate::chat::model_selector::ModelSelector;
+    use crate::chat::chat_line::ChatLine;
+    use crate::chat::shared::ChatAgentAvatar;
+    use crate::chat::shared::ChatModelAvatar;
+    use crate::chat::prompt_input::PromptInput;
 
     ICON_JUMP_TO_BOTTOM = dep("crate://self/resources/icons/jump_to_bottom.svg")
+    ICON_PROMPT = dep("crate://self/resources/icons/prompt.svg")
+    ICON_STOP = dep("crate://self/resources/icons/stop.svg")
```
Comment (noxware) [inline src/chat/chat_panel.rs:36]: These lines are unused here and are not pub. Just a leftover of conflict resolution or I'm missing something?

File: src/chat/model_info.rs [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
-    import makepad_widgets::theme_desktop_dark::*;
-
-    import crate::shared::styles::*;
-    import crate::chat::shared::ChatAgentAvatar;
+    use link::theme::*;
+    use link::shaders::*;
+    use link::widgets::*;
+    
+    use crate::shared::styles::*;
+    use crate::shared::styles::*;
```
Comment (noxware) [inline src/chat/model_info.rs:9]: dup lines

File: src/chat/model_info.rs [hunk 3]
```diff
@@ -27,7 +29,9 @@ live_design! {
         }
     }
 
-    ModelInfo = <View> {
+    pub ModelInfo = <View> {
+        width: Fill,
+        height: Fit,
```
Comment (noxware) [inline src/chat/model_info.rs:34]: both `width` and `height` are dup properties

File: src/landing/landing_screen.rs [hunk 4]
```diff
@@ -5,16 +5,18 @@ use crate::landing::search_bar::SearchBarWidgetExt;
...
-    LandingScreen = {{LandingScreen}} {
+    use link::theme::*;
+    use link::shaders::*;
+    use link::widgets::*;
+
+    use crate::shared::styles::*;
+    use crate::shared::widgets::VerticalFiller;
+    use crate::landing::search_bar::SearchBar;
+    use crate::landing::model_list::ModelList;
+    use crate::landing::sorting::Sorting;
```
Comment (noxware) [inline src/landing/landing_screen.rs:16]: Note that this PR added `Sorting` and `VerticalFiller` but they are unused

**Comment by joulei** [comment]
Thanks for catching those @noxware, they were indeed leftover from conflict resolution.

**Comment by noxware** [comment]
I will merge this now so we can tackle conflicts in other branches ASAP

**Pull request merged**